dpos/manager/dposmanager.go

```diff
@@ -153,6 +161,7 @@ func (d *DPOSManager) Initialize(handler *DPOSHandlerSwitch,
 	d.broadcast = broadcast
 	d.notHandledProposal = make(map[string]struct{})
```

Comment:
Line 162~164, these parameters should initiate in New() method.

Reply:
fixed